### PR TITLE
Replace NavTabs with Tabs and fix url encoding issue in checkboxes

### PR DESCRIPTION
**Pull request by robmunro**
- replace NavTabs with Tabs
- fixed svg encoding issue for checkboxes/radiobuttons in Firefox

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "adslot-ui",
-  "version": "0.1.2",
+  "version": "0.1.4",
```
Comment (omgaz) [inline package.json]: 3?

Comment (robmunro) [inline package.json]: @camjc still hasn't updated the last one ;P

**Comment by omgaz** [comment]
LGTM

**Comment by jamesramsay** [comment]
Weird to start commit message with `- `?

**Comment by robmunro** [comment]
It was for bullet points - but fair enough.

**Comment by omgaz** [comment]
Merging. Need to figure out commit messages for future but not stopping the machine :fist:

**Pull request merged**